openstack/blockstorage/v2/volumes/requests.go

```diff
@@ -122,6 +122,36 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 	})
 }
 
+// ExtendOptsBuilder allows extensions to add additional parameters to the
+// extend action (os-extend) request.
+type ExtendOptsBuilder interface {
```

Comment:
Do we need a separate opts builder for a single argument in this case?  This seems to just add an unnecessary layer.

Reply:
It can seem overly verbose, but it's usually a good idea. At a minimum, for consistency.

Reply:
Agree with @jtopjian :smile: